firmware/src/constants.h

```diff
@@ -261,6 +261,10 @@ typedef enum {
  *
  * Measured in 10ths of a microsecond. The value is from line 2 of Table 3-1
  * in E1.20.
+ *
+ * Note that some responders, like say the Robin 600 Spot, have a very short low
```

Comment:
It tried to keep it reasonably obtuse but still enough info that I can remember how to reproduce it :)